src/app/services/getProjects.js

```diff
@@ -0,0 +1,29 @@
+import http from "@/app/http"
+
+export async function getProjects() {
```

Comment:
same nomenclature question here (as with getDocuments)

Reply:
IDK, I think it works and aligns with CRUD. `get<Noun>`, `create<Noun>`...

Reply:
right, my issue is the overlap between `getProject` and `getProjects` -- they're so close to each other it seems confusing. But maybe I'm overthinking that